LongWarOfTheChosen/Config/XComClassData.ini

```diff
@@ -59,86 +59,71 @@ NumInDeck=0
+							   (AbilityName="SmokeGrenade", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="DeepCover", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="LW_Concentration"), \\
+							   (AbilityName="BringEmOn",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
+							   (AbilityName="SharpshooterAim",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
+							   (AbilityName="DamnGoodGround"), \\
+							   (AbilityName="CloseAndPersonal"), \\
 							   \\
-							   (AbilityName="DamnGoodGround", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="EverVigilant", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="CoveringFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="SharpshooterAim", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="LoneWolf",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
-							   (AbilityName="SmokeGrenade",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
-							   (AbilityName="Flashbanger",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
-							   (AbilityName="Resilience",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
-							   (AbilityName="DeepCover") \\
+							   (AbilityName="PrecisionShot", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="LW_Maim", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="Flashbanger", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="FieldMedic", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="Infighter",  ApplyToWeaponSlot=eInvSlot_Unknown) \\
 					))
 
 ; LWOTC TODO: Add QuickStudy to the above deck if we can think of some way to make it work better in LWOTC
 ;+AWCAbilityTree_Defense=(Level=1, AbilityName="QuickStudy"				)
 
 +RandomAbilityDecks=(DeckName="Tier2_XComAbilities", \\
-					Abilities=((AbilityName="Deadeye", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="DisablingShot", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="WalkFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="GrazingFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="ChainShot", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="IronCurtain", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="CenterMass", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="BringEmOn", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
-							   (AbilityName="LockedOn",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
-							   (AbilityName="Bombard_LW",  ApplyToWeaponSlot=eInvSlot_Unknown), \\
+					Abilities=((AbilityName="HEATWarheads", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="Executioner_LW", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="CyclicFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
```

Comment:
I think cyclic fire is a pretty nice cross class ability to have all in all, and if you want to rework it in the future than removing it from cross class abilities is a few lines of config changes.

Reply:
If you're gonna rework it it already is gonna be disruptive to gunners.
> What makes Cyclic Fire a nice cross class ability

It's unique way to give an immobile but massive boost in fire power, allowing to shoot thrice. It's not that good on gunners because they can get a combination of traverse + RF, but cyclic is one ability, and that matters for the AP tree a lot. It's a powerful but sitiational ability, making it a perfect fit.
Maybe you can create a new ability for gunners and rename cyclic fire to something else.